packit_service/worker/helpers/build/copr_build.py

```diff
@@ -333,29 +382,26 @@ def check_if_custom_copr_can_be_used_and_report(self) -> bool:
 
         :return: True if the matching is configured.
         """
-        configured_copr_project = f"{self.job_owner}/{self.job_project}"
-        if (
-            forge_project := (
-                f"{self.project.service.hostname}/{self.project.namespace}/{self.project.repo}"
-            )
-        ) not in (
-            allowed_projects := self.service_config.allowed_forge_projects_for_copr_project.get(
-                configured_copr_project, []
-            )
-        ):
-            logger.warning(
-                f"{forge_project} can't use {configured_copr_project} "
-                f"(Only {allowed_projects} are allowed.)"
-            )
-            self.report_status_to_build(
-                description=CUSTOM_COPR_PROJECT_NOT_ALLOWED_STATUS.format(
-                    copr_project=configured_copr_project
-                ),
-                state=BaseCommitStatus.neutral,
-                markdown_content=CUSTOM_COPR_PROJECT_NOT_ALLOWED_CONTENT.format(
-                    copr_project=configured_copr_project
-                ),
-            )
+        if not self.is_forge_project_allowed_to_build_in_copr():
+            # TODO: to be removed in the future (if statement)
+            if self.is_forge_project_allowed_to_build_in_copr_by_config():
+                markdown_content = CUSTOM_COPR_PROJECT_ALLOWED_IN_PACKIT_CONFIG.format(
+                    copr_project=self.configured_copr_project,
+                    forge_project=self.forge_project,
+                )
+                self.status_reporter.comment(body=markdown_content)
+                return True
+            else:
+                self.report_status_to_build(
+                    description=CUSTOM_COPR_PROJECT_NOT_ALLOWED_STATUS.format(
+                        copr_project=self.configured_copr_project
+                    ),
+                    state=BaseCommitStatus.neutral,
+                    markdown_content=CUSTOM_COPR_PROJECT_NOT_ALLOWED_CONTENT.format(
+                        copr_project=self.configured_copr_project,
+                        forge_project=self.forge_project,
+                    ),
+                )
             return False
         return True
```

Comment:
Maybe a bit more readable when done like this (not sure if the format will match black..;):

```suggestion
        if self.is_forge_project_allowed_to_build_in_copr():
            return True

        # TODO: to be removed in the future (if statement)
        if self.is_forge_project_allowed_to_build_in_copr_by_config():
            markdown_content = CUSTOM_COPR_PROJECT_ALLOWED_IN_PACKIT_CONFIG.format(
                copr_project=self.configured_copr_project,
                forge_project=self.forge_project,
            )
            self.status_reporter.comment(body=markdown_content)
            return True

        self.report_status_to_build(
            description=CUSTOM_COPR_PROJECT_NOT_ALLOWED_STATUS.format(
                copr_project=self.configured_copr_project
            ),
            state=BaseCommitStatus.neutral,
            markdown_content=CUSTOM_COPR_PROJECT_NOT_ALLOWED_CONTENT.format(
                copr_project=self.configured_copr_project,
                forge_project=self.forge_project,
            ),
        )
        return False
```